models/setup.go

```diff
@@ -0,0 +1,33 @@
+package models
+
+import (
+	"fmt"
+	"gorm.io/driver/postgres"
+	"gorm.io/gorm"
+	"os"
+)
+
+var DB *gorm.DB
+
+func ConnectDataBase() {
+	dsn := fmt.Sprintf("host=%s user=%s port=%s sslmode=%s", os.Getenv("DB_HOST"), os.Getenv("DB_USER"), os.Getenv("DB_PORT"), os.Getenv("DB_SSL"))
+
+	db, err := gorm.Open(postgres.Open(dsn), &gorm.Config{})
+
+	if err != nil {
+		panic("Failed to connect to database!")
+	}
+
+	db.AutoMigrate(&Cafe{})
```

Comment:
Will this line run the migration every time the program is run? Is that a problem if so?